Cython/Utility/ObjectHandling.c

```diff
@@ -2317,7 +2317,11 @@ static CYTHON_INLINE PyObject* __Pyx_PyObject_FastCallDict(PyObject *func, PyObj
     #endif
 
     #if CYTHON_VECTORCALL
+    #if PY_VERSION_HEX >= 0x030D0000
+    vectorcallfunc f = PyVectorcall_Function(func);
+    #else
     vectorcallfunc f = _PyVectorcall_Function(func);
+    #endif
```

Comment:
This can probably be done better with a `__Pyx_PyVectorcall_Function()` macro, in order to avoid doing this check wherever we use the function.

Reply:
We may as well make the define 0x03080000 too and use the official name as far back as we can.